src/fu-tool.c

```diff
@@ -2935,8 +2935,9 @@ fu_util_security(FuUtilPrivate *priv, gchar **values, GError **error)
 		g_set_error_literal(error,
 				    FWUPD_ERROR,
 				    FWUPD_ERROR_NOT_SUPPORTED,
-				    "Not enough data was provided to make an HSI calculation. "
-				    "To ignore this warning, use --force.");
+				    "Not enough data was provided to make an HSI calculation.\n"
+				    " » https://fwupd.github.io/hsi.html#not-enough-info\n"
+				    "To ignore this warning, use --force");
```

Comment:
I think we should translate these strings.

Reply:
fixed